README.md

```diff
@@ -469,15 +488,26 @@ module.exports = {
 
 ### `additionalData`
 
-Type: `String|Function`
+Type:
+
+```ts
+type additionalData =
+  | string
+  | ((
+      content: string | Buffer,
+      loaderContext: LoaderContext,
+      meta: any
```

Comment:
We don't have `meta` here, only content and `loaderContext`

Reply:
Changed